bin/pint_db.py

```diff
@@ -282,7 +281,7 @@ def orm_update_tables(db, provider, tables, version):
 
 
 def orm_load_database(pint_data, db_logfile=None):
-    db = init_db(outputfile=db_logfile)
+    db = init_db(outputfile=db_logfile, create_all=False)
```

Comment:
Question: Should the default not now be to not create the tables, which the requirement being that any tooling that wants to skip the versioned DB setup is the one to specify create_all=True when calling init_db().

The default should probably be based upon the production scenario, in which case the DB should be being initialised using this pint_db.py tool, and the application side of things should be starting up pointing at an already initialised DB, and therefore shouldn't need to create any tables...

Reply:
I don't think the ORM should be creating tables as it doesn't do sane schema migration. That's the job of the schema migration layer. The reason I set the default to `True` was to preserve backward compatibility. Even in production scenario, schema creation and data population should come before the app.